NAMESPACE

```diff
@@ -17,14 +18,18 @@ export(.vsc.getCompletion)
 export(.vsc.getCustomInfo)
 export(.vsc.getVarInfo)
 export(.vsc.getVarLists)
+export(.vsc.handleJson)
 export(.vsc.isEvaluating)
 export(.vsc.listVarInfo)
+export(.vsc.listenOnPort)
 export(.vsc.onError)
 export(.vsc.print)
 export(.vsc.removeVarInfo)
 export(.vsc.resetVarInfo)
 export(.vsc.sendToVsc)
 export(.vsc.setBreakpoints)
 export(.vsc.setStoredBreakpoints)
+export(getSkipFromBottom)
```

Comment:
Are these two functions necessary to export?

Reply:
Nope, just forgot to remove them after debugging, thanks.

Reply:
Fixed in https://github.com/ManuelHentschel/vscDebugger/pull/67